<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PickActorWindow</class>
 <widget class="QWidget" name="PickActorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>847</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Config</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="4" column="2">
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Source</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Bounds</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="2" column="1">
               <widget class="QDoubleSpinBox" name="yMinSpinBox">
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>X1</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Z1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QDoubleSpinBox" name="xMinSpinBox">
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Y1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>X2</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDoubleSpinBox" name="zMinSpinBox">
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Z2</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Y2</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QDoubleSpinBox" name="xMaxSpinBox">
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QDoubleSpinBox" name="yMaxSpinBox">
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QDoubleSpinBox" name="zMaxSpinBox">
                <property name="minimum">
                 <double>-100.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="sourceCombo">
           <item>
            <property name="text">
             <string>Cone</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Sphere</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cube</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Arrow</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Cylinder</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Import File</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Export File</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="importFileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="exportFileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="importButton">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Preview</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="previewLayout"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
